tests/test_als.py

```diff
@@ -1,8 +1,8 @@
 import unittest
 
 import numpy as np
-import pandas as pd
 from scipy.sparse import csr_matrix
+from numpy import genfromtxt
```

Comment:
This import can be removed and call `genfromtxt` as `np.genfromtxt`, as the rest of numpy calls.